crawl-ref/source/makeitem.cc

```diff
@@ -547,15 +547,11 @@ static special_missile_type _determine_missile_brand(const item_def& item,
     case MI_JAVELIN:
         rc = random_choose_weighted(30, SPMSL_RETURNING,
                                     32, SPMSL_PENETRATION,
-                                    32, SPMSL_POISONED,
-                                    21, SPMSL_STEEL,
                                     20, SPMSL_SILVER,
                                     nw, SPMSL_NORMAL);
         break;
     case MI_TOMAHAWK:
-        rc = random_choose_weighted(15, SPMSL_POISONED,
-                                    10, SPMSL_SILVER,
-                                    10, SPMSL_STEEL,
+        rc = random_choose_weighted(10, SPMSL_SILVER,
```

Comment:
These changes will lower the total weight of a branded tomahawk / javelin. Not sure if this is intentional; in fact it might be ok, but it also could make sense to distribute the weight over the remaining brands. (Some other dev might want to weight in.)